src/RouterSelector.native.js

```diff
@@ -38,6 +40,7 @@ let AppRouter = React.lazy(() => {
     retryImport(() => import(/* webpackChunkName: "router" */ './Router')),
     initAnalyticsAndFireAppOpen(),
     delay(animationDuration),
+    AsyncStorage.setItem(OLD_NOTIFICATIONS, []),
```

Comment:
1. what is the purpose of OLD_NOTIFICATIONS?
2. here you set it as array but in other places you actually set it as feed.id
3. why not properties?

Reply:
1. For now logic is next: every bg-fetch call (every 15 min) we get last 20 feed items. On first call we mark all notifications as old. On next call we again get last 20 items, and if there are items not marked as old, we send notification with this item and mark it as old. So by marking as "old" we exclude item from notification flow

Reply:
I dont understand. also you didnt answer 2+3
@serdiukov-o-nordwhale

Reply:
2 and 3 fixed